src/styles/blocks/calendar.scss

```
@@ -0,0 +1,44 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: #{$dayGap};
+  width: getWidth($daySize, $dayGap);
+  margin: 10px auto;
```

Don't be greedy, add some top `paddings`
```suggestion
  padding: 10px;
```
or
```suggestion
  padding: 10px 0;
```